src/sentry/lang/native/plugin.py

```diff
@@ -443,5 +438,9 @@ def report_error(e):
 class NativePlugin(Plugin2):
     can_disable = False
 
-    def get_event_preprocessors(self, **kwargs):
-        return [preprocess_apple_crash_event, resolve_frame_symbols]
+    def get_event_preprocessors(self, data, **kwargs):
+        if data.get('sentry.interfaces.AppleCrashReport'):
```

Comment:
You mean instead of it being a generator? Is there a problem with me changing the docstring to say it should return an iterable instead? This isn't used elsewhere, so it seems safe that we can change the definition slightly to better accommodate.